vivisect/tools/graphutil.py

```diff
@@ -612,6 +625,33 @@ def getGraphNodeByVa(fgraph, va):
             return nva
     return None
 
+def findRemergeDown(graph, va):
```

Comment:
track through a graph to where it remerges after a split.  very useful for compartmentalizing code chunks.